### ISSUE TITLE
macOS guests with paravirtualized graphics on Intel

### ISSUE THREAD

**Issue by vit9696** [utmapp/UTM#3491]
It will be great to support macOS guests on Intel CPUs, not just on ARM. For that we can utilise OpenCore. Apple does provide paravirtualized Metal adapter (https://developer.apple.com/documentation/paravirtualizedgraphics) on both ARM and x86, so these VMs can also be quite performant starting with macOS 11.

**Comment by osy** [comment]
Currently Virtualization.framework only supports graphics on macOS 12 ARM64 https://developer.apple.com/documentation/virtualization/vzmacgraphicsdisplayconfiguration and QEMU does not support ParavirtualizedGraphics. For the latter, we can look into implementing it if there’s a clear advantage over the existing virtiogpu adapter which has plenty of support.

**Comment by vit9696** [comment]
Yes, macOS is a **lot** better with it. VMware and Parallels both support it, but for VMware the support is experimental, and is partially broken at the moment:
- https://kb.vmware.com/s/article/81657
- macOS recovery/installer do not start with paravirtualised graphics (only the installed OS does)
- https://twitter.com/mikeroySoft/status/1456732256941187075
